catalog/spatial/registry/registry-schema-bindings/pom.xml

```diff
@@ -154,29 +184,33 @@
                         </goals>
                         <configuration>
                             <haltOnFailure>true</haltOnFailure>
+                            <includes>
+                                <include>**/RegistryPackageUtils.*</include>
+                                <include>**/RegistryPackageWebConverter.*</include>
+                            </includes>
                             <rules>
                                 <rule>
                                     <element>BUNDLE</element>
                                     <limits>
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.0</minimum>
+                                            <minimum>0.90</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.0</minimum>
+                                            <minimum>0.65</minimum>
```

Comment:
The branch and complexity coverages are too low. Any way we can either simplify the code to reduce the number of branches and/or complexity? If not, we should check what paths aren't covered and add more unit tests.